src/Components/ElectionProtection/ElectionProtection.js

```diff
@@ -1,33 +1,55 @@
+import { FormattedMessage } from 'react-intl';
 import './ElectionProtection.css';
 
 const ElectionProtection = () => {
   return (
     <div className="election-protection-container">
-      <h2 className="election-protection-title">Election Protection</h2>
-      <h3 className="election-protection-sub-title">Protect your right to vote by following these basic guidelines:</h3>
+      <h2 className="election-protection-title">
+        <FormattedMessage
+          id="electionProtection.title"
+          defaultMessage="Election Protection" />
+      </h2>
+      <h3 className="election-protection-sub-title">
+        <FormattedMessage
+          id="electionProtection.subtitle"
+          defaultMessage="Protect your right to vote by following these basic guidelines:" />
+      </h3>
       <article className="article-content">
         <ol>
           <li className="list-item">
-            Before heading out to the polls, make sure to check what forms of ID will be required in order to be able to vote in your specific state. You can verify the valid forms of ID that are accepted in your state
+            <FormattedMessage 
+              id="electionProtection.listItem1"
+              defaultMessage="Before heading out to the polls, make sure to check what forms of ID will be required in order to be able to vote in your specific state. You can verify the valid forms of ID that are accepted in your state"
+            />
               <a className="link-to-voter-id-laws"
                 href="https://www.vote.org/voter-id-laws/"
                 target="_blank"
                 rel="noopener"
                 aria-describedby="new-window-2">
-                here
-              </a>.
+                <FormattedMessage
+                  id="electionProtection.hereLink"
+                  defaultMessage="here" />
+              </a>.  
           </li>
```

Comment:
I'll be completely honest, I tried (and eventually succeeded) to pull all of this static text into an external file, but handling the link really tripped me up.  Honestly, I was not a fan of the solution, and I would love to see how someone else would approach it.  Maybe in some sort of Capstone Project reflection time?